src/swig/stir.i

```diff
@@ -640,9 +640,7 @@ namespace std {
       //      {
       //        num_sinos += 2*proj_data.get_num_axial_poss(s);
       //      }
-      int num_sinos = proj_data.get_num_sinograms();
-
-      Array<4,float> array(IndexRange4D(proj_data.get_num_tof_poss(),num_sinos, proj_data.get_num_views(), proj_data.get_num_tangential_poss()));
+      Array<4,float> array(IndexRange4D(proj_data.get_num_tof_poss(),proj_data.get_num_segments(), proj_data.get_num_views(), proj_data.get_num_tangential_poss()));
```

Comment:
incorrect. This would have to be
```c++
const int num_non_tof_sinos = proj_data.get_num_sinograms() / proj_data.get_num_tof_poss();
```
and then use that. I'll fix this.

Reply:
t.b.h. then I do not like this. 
We have to make sure that under any situations the ```get_num_for_poss()``` returns zero. 
Which currently is the default value for the creation of a ```ProjDataInfo```....

Reply:
I've done this slightly differently (by introducing `get_num_non_tof_sinograms`), but `get_num_tof_poss()` should never be zero of course. I guess we'd need a test for this then.

I don't understand what you mean with "The mashing factor is zero" (but possibly it's no longer relevant)